src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail-form/dot-apps-configuration-detail-form.component.ts

```diff
@@ -32,6 +41,10 @@ export class DotAppsConfigurationDetailFormComponent implements OnInit {
         });
 
         this.emitValues();
+
+        setTimeout(() => {
+            this.formContainer.nativeElement.querySelector(`#${this.formFields[0].name}`).focus();
```

Comment:
https://angular.io/api/core/ViewChild ???

Reply:
the fields of that form are created dynamically, so there's no way to set a tag (viewChild) to grab the first element